unsymlink-lib

```diff
@@ -164,7 +164,11 @@ class MigrationState(object):
         self.eroot = eroot
 
     def analyze(self, usr_merge, real_prefixes):
-        from portage import create_trees, _encodings
+        from portage import create_trees, const, _encodings
+
+        eprefix = const.EPREFIX.encode(_encodings['fs'])
```

Comment:
I don't think this is entirely safe. I would have preferred if you moved `create_trees()` earlier, and grabbed it from the config instance.

Reply:
Agreed, EPREFIX could be overridden by environment and portage.const.EPREFIX is always a constant.